ush/python/pygfs/task/marine_analysis.py

```diff
@@ -54,50 +52,47 @@ def __init__(self, config):
         else:
             _berror_model = 'marine_background_error_static_diffusion'
 
+        # Get restart date
+        if self.task_config.DOIAU:
+            _rst_date = to_fv3time(self.task_config.WINDOW_BEGIN)
+            _cice_rst_date = to_fv3time(self.task_config.WINDOW_BEGIN)
+        else:
+            _rst_date = to_fv3time(self.task_config.current_cycle)
+            _cice_rst_date = to_fv3time(self.task_config.current_cycle)
```

Comment:
```suggestion
        if self.task_config.DOIAU:
            _rst_date = to_fv3time(self.task_config.WINDOW_BEGIN)
        else:
            _rst_date = to_fv3time(self.task_config.current_cycle)
```
`_cice_rst_date` is the same as `_rst_date`, so don't really need this.

Reply:
`rst_date` and `cice_rst_date` are indeed the same here. @guillaumevernieres Do we ever expect these to be different?

Reply:
@DavidNew-NOAA , _probably_ not

Reply:
Add a TODO for now and deal with this later.